developers/integration.rst

```diff
@@ -108,65 +108,31 @@ for worked examples of each transaction type.
 
 .. _integration-instantsend:
 
-InstantSend Overview
-====================
+InstantSend
+===========
 
 This documentation is also available as a `PDF <https://github.com/dashpay/docs/raw/master/binary/integration/Dash_v0.13_InstantSend.pdf>`__.
 
 InstantSend is a feature provided by the Dash network that allows for
-zero-confirmation transactions to be safely accepted by Merchants and
-other service providers. All InstantSend Transactions are secured for 25
-blocks by the “Masternode Network” at the moment of broadcast. The
-transaction is mined into the next block in accordance with standard
-blockchain principles.
+zero-confirmation transactions to be safely accepted and re-spent. All
+valid transactions are secured by the Dash network at the moment of
+broadcast. Every secured transaction is included in the next block in
+accordance with standard blockchain principles.
 
 InstantSend is enabled by the Masternode Network which comprises
-approximately 4,800 masternode servers. These nodes are differentiated
-from standard nodes by having proven ownership of 1,000 Dash. One
-responsibility that is appointed to this special type of server is to
-perform "Transaction Locking", also known as InstantSend.
-
-This concept works as an extension to network consensus. When an
-"InstantSend" transaction occurs the network goes through an extra
-validation process which examines the following two properties of the
-transaction:
-
-1. **Input Maturity:** the network will require all inputs to have at
-   least 6 confirmations.
-
-2. **Input Composition:** the number of inputs in use dictates fee 
-   requirements.
-
-   a. **1 - 4 inputs:** per-kB fee of 0.00001 DASH using Automatic 
-      InstantSend.
-   b. **5+ inputs:** per-input fee of 0.0001 DASH is required.
-
-Assuming the **Input Maturity** and **Input Composition** requirements
-are met, the network will "lock" the inputs related to this transaction
-for 25 blocks. Transactions carrying 4 or fewer inputs are referred to
-as a “simple transaction” and carry no extra fee.
-
-
-Automatic InstantSend
----------------------
+approximately 5,000 masternode servers. These nodes are differentiated
+from standard nodes by having proven ownership of 1,000 Dash, making the
+network `highly resistant to Sybil attacks <https://en.wikipedia.org/wiki/Sybil_attack>`__. 
+Masternodes form `Long-Living Masternode Quorums (LLMQs) <https://github.com/dashpay/dips/blob/master/dip-0006.md>`__, 
+which are responsible for providing instant certainty to the transaction
+participants that the transaction inputs cannot be respent, and that the
+transaction will be included in the next block instead of a conflicting
```

Comment:
There is a guarantee that a conflicting tx will not be mined, but there is _not_ a guarantee that locked txs will appear in the next block.

Reply:
```suggestion
transaction will be included in a following block instead of a conflicting
```
alright like this?